src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+	<head>
+	<link type="text/css" rel="stylesheet" href="style1.css" media="screen">
+		<meta charset="utf-8">
+		<Title></title>
+		<script type="text/javascript" src="audio.js"></script>
+		<script type="text/javascript" src="main.js"></script>
+	</head>
+	<body onload="init()">
+	</body>

```

Comment:
Creating your entire DOM tree with JS is bad practice.
Consider setting up the page skeleton in this `index.html` and manipulating it via JS later on.